### PR TITLE
docs: add body-parser

### LINKED ISSUE #160: bodyparser -> milliparsec

### PR DESCRIPTION
**Pull request by Namchee**
## Overview

Closes #160.

This pull request adds section for `body-parser` and its alternative (namely, `milliparsec`)

### CODE REVIEW

File: docs/modules/README.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ ESLint plugin.
 
 - [`@jsdevtools/ez-spawn`](./process-exec.md)
 - [`bluebird` / `q`](./bluebird-q.md)
+- [`body-parser`](./body-parser.md)
+- [`builtin-modules`](./is-builtin-module.md)
```
Comment (43081j) [inline docs/modules/README.md:18]: Did you mean to add this line?

Was it missing before?

Comment (Namchee) [reply docs/modules/README.md:18]: Yes, they're surprisingly not available before.

Comment (43081j) [reply docs/modules/README.md:18]: ah it exists u nder `is-builtin-module` here:

https://github.com/es-tooling/module-replacements/blob/7d864c64f68394ea55f33ef67c712c217e5e68c4/docs/modules/README.md?plain=1#L30

Comment (Namchee) [reply docs/modules/README.md]: Oops, I missed that since I thought the entries were lexicographically sorted by the markdown name. Removed.

**Review by 43081j — APPROVED** [review]
looks good to me, but lets get at least one other opinion before we merge 👍

File: docs/modules/body-parser.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Body-parser
+
+`body-parser` can be replaced with more modern alternatives which is both lighter and faster.
```
Comment (SuperchupuDev) [inline docs/modules/body-parser.md:3]:
```suggestion
`body-parser` can be replaced with more modern alternatives which are both lighter and faster.
```
small grammar mistake

**Comment by TheThing** [comment]
Discussion on a discord server revealed that we should probably wait until default post limit is implemented into millisec. Don't want to give malicious people the ability to DDoS a server from this recommendation.

**Comment by 43081j** [comment]
It may also be worth explaining that some simple use cases don't need a package

Maybe we list "native" as an alternative and give some examples?

**Review by 43081j — APPROVED** [review]

**Pull request merged**